templates/bot/ts/notification-trigger-http/notifyHttpTrgger/function.json

```diff
@@ -16,5 +16,5 @@
       "name": "res"
     }
   ],
-  "scriptFile": "../dist/httpTrigger.js"
+  "scriptFile": "./dist/httpTrigger.js"
```

Comment:
May be the `scriptFile` is still `../dist/httpTrigger.js`

Reply:
Fixed.